hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/HawkbitCommonUtil.java

```diff
@@ -85,11 +85,10 @@ public static boolean isNotNullOrEmpty(final Map<?, ?> mapToCheck) {
      *         the text is not empty.
      */
     public static String trimAndNullIfEmpty(final String text) {
```

Comment:
Probably can remove this method and use springs `StringUtils.isEmpty()` for that? As already used.